tests/unit/test_environment.py

```diff
@@ -8,109 +7,106 @@
+    example_plain_env.set_location(latitude, longitude)
+    assert example_plain_env.latitude == latitude
+    assert example_plain_env.longitude == longitude
 
 
-def test_set_elevation(example_env):
-    """Tests if the elevation is set correctly in the environment object. This
-    one basically set the elevation using the set_elevation() method and then
-    checks if the elevation is the same as the one in the example environment
-    object.
+@pytest.mark.parametrize("elevation", [(-200), (0), (200)])
+def test_elevation_set_elevation_saves_elevation(elevation, example_plain_env):
+    """Tests elevation is set correctly in the environment obj.
 
     Parameters
     ----------
-    example_env : rocketpy.Environment
-        Example environment object to be tested.
+    example_plain_env : rocketpy.Environment
     """
-    example_env.set_elevation(elevation=200)
-    assert example_env.elevation == 200
+    example_plain_env.set_elevation(elevation=elevation)
+    assert example_plain_env.elevation == elevation
 
 
-def test_set_topographic_profile(example_env):
-    """Tests the topographic profile in the environment object.
+@pytest.mark.parametrize(
+    "latitude, longitude, theoretical_elevation", [(46.90479, 8.07575, 1565)]
+)
+def test_location_set_topographic_profile_computes_elevation(
+    latitude, longitude, theoretical_elevation, example_plain_env
+):
+    """Tests elevation computation given topographic profile in the environment obj.
 
     Parameters
     ----------
-    example_env : rocketpy.Environment
-        Example environment object to be tested.
+    example_plain_env : rocketpy.Environment
+    latitude: float
+        The latitude in decimal degrees.
+    longitude: float
+        The longitude in decimal degrees.
     """
-    example_env.set_location(46.90479, 8.07575)
-    example_env.set_topographic_profile(
+    example_plain_env.set_topographic_profile(
         type="NASADEM_HGT",
         file="data/sites/switzerland/NASADEM_NC_n46e008.nc",
         dictionary="netCDF4",
     )
-    assert (
-        example_env.get_elevation_from_topographic_profile(
-            example_env.latitude, example_env.longitude
-        )
-        == 1565
+    computed_elevation = example_plain_env.get_elevation_from_topographic_profile(
+        latitude, longitude
     )
+    assert computed_elevation == theoretical_elevation
 
 
-def test_export_environment(example_env_robust):
+def test_environment_export_environment_exports_environment_json(example_spaceport_env):
     """Tests the export_environment() method of the Environment class.
 
     Parameters
     ----------
-    example_env_robust : rocketpy.Environment
-        Example environment object to be tested.
+    example_spaceport_env : rocketpy.Environment
     """
-    assert example_env_robust.export_environment(filename="environment") == None
+    assert example_spaceport_env.export_environment(filename="environment") == None
```

Comment:
This is more pythonic:

```suggestion
    assert example_spaceport_env.export_environment(filename="environment") is None
```

Reply:
It happens in more sections of this same file.

Reply:
I understand your argument, and honestly I didn't know the difference between `==` and `is` so much. The stackoverflow link really helped.

There's a "programming recommendation" in PEP8: "Comparisons to singletons like None should always be done with is or is not, never the equality operators". See [here](https://peps.python.org/pep-0008/#programming-recommendations).

Addinionally, there's a short thread [here](https://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or) and [here](https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none).

This is one of those things in Python where, no matter what we choose, code will be working.

If you still want to keep the `==`, no problems at this point. There are many parts of the library and tests where we use `== None`. The first reason for me to comment on this is due to pylint warnings: [C0121](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/singleton-comparison.html)

Reply:
Btw these comments are also valid for the `== True` occurrences

Reply:
For your information, @phmbressan .
I think you'd enjoy reading these threads eventually.

Reply:
Hmmmm I didn't know `None` was a singleton, now it makes sense. Then I agree `is` should be used instead. I will push a correction